dependencies/src/test/java/com/google/cloud/tools/opensource/dependencies/RepositoryUtilityTest.java

```diff
@@ -80,9 +81,13 @@ public void testReadBom_path() throws MavenRepositoryException, ArtifactDescript
     if (currentArtifacts.size() != 217) {      
       Set<Artifact> current = Sets.newHashSet(currentArtifacts);
       Set<Artifact> old = Sets.newHashSet(oldArtifacts);
-      SetView<Artifact> added = Sets.difference(current, old);
+      
+      Set<String> currentKeys = current.stream().map(Artifacts::makeKey).collect(Collectors.toSet());
```

Comment:
Why not get the set of strings in one step?

```java
Set<String> current = currentArtifacts.stream().map(Artifacts::makeKey).collect(toSet());
```

Reply:
done